tests/config/woodpecker/ldap/20_users.ldif

```diff
@@ -0,0 +1,74 @@
+dn: uid=alice,ou=users,dc=opencloud,dc=eu
+objectClass: inetOrgPerson
+objectClass: organizationalPerson
+objectClass: person
+objectClass: top
+uid: alice
+givenName: Alice
+sn: Hansen
+cn: alice
+displayName: Alice Hansen
+description: Senior DevOps engineer responsible for cloud infrastructure automation.
+mail: alice@example.org
+departmentNumber: cd22ea13-f6b4-4f5f-a2c2-69b5a0f07a1a
```

Comment:
another nit-pick: why use an uuid here? For tests a simple string should be sufficent (e.g.: tenant1), makes it easier to maintain/read